content/src/controller/Controller.ts

```diff
@@ -540,14 +540,18 @@ export class Controller {
     }
 
     try {
-      await this.denylist.addTarget(target, { timestamp, authChain })
-      res.status(201).send()
+      const result: OperationResult = await this.denylist.addTarget(target, { timestamp, authChain })
+      if (isSuccessfulOperation(result)) {
+        res.status(201).send()
+      } else {
+        res.status(400).send(result.message)
```

Comment:
Didn't you have to change existing tests with this change? Didn't we have a test that verified when a denylist action failed?

Reply:
It wasn't an API test, it just checked that the service failed.